recipes/icon/LICENSE.txt

```diff
@@ -0,0 +1,39 @@
+MIT License
+
+Copyright (c) 2021 Arthur Eschenlauer
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```

Comment:
Where is this coming from?

Reply:
@isuruf Thank you for reviewing my PR!
Thank you. I am not sure what you are asking in this comment.

The MIT license is decades old and frequently used on GitHub.
https://spdx.org/licenses/MIT.html

My intention here is to provide a license for the package (not for Icon, which is public domain).  If there is another way that I should be approaching this (up to and including not providing a license file), please provide advice.  Thank you.

Reply:
See also:
```
sed -e "1,/Note Well/d" ${RECIPE_DIR}/LICENSE.txt > ${PREFIX}/icon/LICENSE.txt
```

Again, I am not asserting that I am handling this well; this was how I was trying to achieve some valid-ish licensing, but I am totally open to guidance!  Thank you so much.

Reply:
> My intention here is to provide a license for the package

We require all our contributors to agree to the BSD-3-Clause for the recipe itself. https://github.com/conda-forge/staged-recipes/blob/master/LICENSE.txt

There's no need to add your name or copyright here.

Reply:
@isuruf Okay, I agree.  Does that mean that I put this into the `license` and `license-family` fields of `meta.yaml` and put that `LICENSE.txt` into my recipe?